src/server/controllers/movie.js

```diff
@@ -13,14 +13,34 @@ const getAllMovies = async (req, res) => {
 const createMovie = async (req, res) => {
     const { title, description, runtimeMins } = req.body;
 
+    const authorization = req.headers['authorization']
```

Comment:
Hi, what does this authorization variable do?

Reply:
If you have a look at line 72, you can get the value of the requested authorised token.